### PR TITLE
[22987] [Onboarding] Getting started video screen (2)

### PR DESCRIPTION
**Pull request by HDinger**
This includes a modal with onboading infos for first time users. 
##### Acceptance criteria:
- [x] This is the first screen for an admin that logs in for the first time to his organization.
- [x] It open a standard model window.
- [x] The user can close the modal window by clicking th "x" in the top right corner.
- [x] Styling according to the visual

_Header_
- [x] The gravatar icon is shown in the header on the left.
- [x] Welcome header

_Main section_
- [x] Vimeo video and text

_Footer_
- [x] "Next" Button"

https://community.openproject.com/work_packages/22987/activity

This modal is based on the modal of foundation for apps. This why it is based on the PR which includes foundation.js: https://github.com/opf/openproject/pull/4447

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hdinger-feature-22987-onboarding-starting-video-foundation.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://hdinger-feature-22987-onboarding-starting-video-foundation.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

**Comment by ulferts** [comment]
@HDinger, @furinvader I don't think that using foundation 1.2 is a wise decision at this point in time as the [changelog states that it includes an update to angular 1.4](https://github.com/zurb/foundation-apps/blob/master/CHANGELOG.md#general).

This is probably the reason for the problem bower faces when executing the tests:

```
bower angular-animate              resolution Unsuitable resolution declared for angular-animate: 1.3.14
```

**Comment by HDinger** [comment]
@ulferts @furinvader I changed the version of foundation-apps to 1.1.0 again. 
@furinvader What do we do with the spec that was failing with the older version?

**Comment by oliverguenther** [comment]
I'll take a look at the failing test

File: frontend/tests/unit/tests/layout/query-menu-item-factory-test.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ describe('queryMenuItemFactory', function() {
     // The document does not seem to be cleaned up after each test instead each
     // test leaves additional DOM. Thus the tests are not independent.
     // Therefore we clean it by hand.
-    $document.find('body').html('');
+    menuContainer.remove();
```
Comment (oliverguenther) [inline frontend/tests/unit/tests/layout/query-menu-item-factory-test.js:78]: @furinvader @ulferts 

FYI: Karma really doesn't clean up the DOM so removing the element is fine. Clearing the body, on the other hand, will remove all custom assets (such as the CSS I required above), and thus fail other, completely unrelated tests.

We need to be careful what we remove. It should be fine to keep a reference to the element(s) we added and remove them afterwards.

**Comment by oliverguenther** [comment]
@HDinger Looks good to me , and the test should be green now. I'm wondering if the `next` is the correct label for now? Or are you planning to add a second modal in this PR?

**Comment by HDinger** [comment]
@oliverguenther I agree that this is confusing. This was Niels suggestion, but I guess because he thought the other steps will be there soon. I guess we can change this for now and undo it later.

**Comment by oliverguenther** [comment]
Nah, let's not do that. If it's expected to gain more steps with 5.1. this is okay for me.

**Comment by HDinger** [comment]
ok, that is fine for me

**Pull request merged**